### PR TITLE
Django 1.8

### PR DESCRIPTION
**Pull request by glogiotatidis**
django 1.8 magic!

### CODE REVIEW

**Comment by glogiotatidis** [comment]
needs https://github.com/niwibe/django-jinja/pull/114

File: {{ cookiecutter.project_name }}/requirements.txt [hunk 1]
```diff
@@ -55,14 +51,6 @@ django-csp==2.0.3
 # sha256: k_gRIthpg8uwVs0Syn3jnXYLbz4wYOQ-_1xeIDRRb6I
 pytz==2015.2
 
-# sha256: eLAxFrrdDbsu1N7clqSj1CE4uUu4n8Lrmffzvf0Zn1Y
-# sha256: zIrr3sWl-5iZEvFX93s8IaXi8tpiOvkKe0drEGqDSr8
-# sha256: drxjpOLV5aDfd8p9GPD1bixGz7YrcRA7qSqSx5-rHgM
-nose==1.3.4
```
Comment (jgmize) [inline {{ cookiecutter.project_name }}/requirements.txt]: This causes a [test failure](https://travis-ci.org/mozilla/sugardough/jobs/57331269) but I'm not opposed to removing it if you want to replace the `eq_` shorthand we get from `nose.tools`.

**Comment by peterbe** [comment]
so pytest is awesome and all but is it able to keep database tables like django-nose can do?
So if you have a big test suite and you're re-running just a single test repeatedly, won't that suddenly become much slower when you don't use django-nose?

**Comment by jgmize** [comment]
> so pytest is awesome and all but is it able to keep database tables like django-nose can do?
> So if you have a big test suite and you're re-running just a single test repeatedly, won't that suddenly 
> become much slower when you don't use django-nose?

@peterbe yes, if you use https://pypi.python.org/pypi/pytest-django/ -- this PR doesn't specifically include pytest-django in the template though, so the default would be to use the standard test runner. We do use it on https://github.com/mozilla/masterfirefoxos, which was derived from an older version of the sugardough template, and I think it would be a good default, but it doesn't necessarily need to be part of this PR.

**Comment by peterbe** [comment]
@jgmize My point is that if we upgrade to Django 1.8 now it appears we have to drop django-nose because django-nose isn't friends with django 1.8 yet. So if we upgrade to 1.8 we lose the productivity boost that django-nose is. I don't have a particular strong argument or stake in the game, just trying to raise a debate. 

Truth be told, perhaps we'll never win if we try to be too specific. Productivity-nerds who want/need django-nose will have to get their own act together to patch django-nose or some day later retrofit django-nose in when it starts to work together again. And then there are good reasons to allow for the many benefits of pytest and people who prefer that. 

Maybe by staying out of the game of a better test-runners we win. What do you think?

**Comment by glogiotatidis** [comment]
> Maybe by staying out of the game of a better test-runners we win. What do you think?

Yes maybe that's the way to go: We remove nose and we stay with vanilla django test. We can open an issue to discuss this further after we upgrade to django 1.8

**Comment by glogiotatidis** [comment]
I believe this is ready. r?

**Comment by jgmize** [comment]
r+
Nice work, @glogiotatidis

**Pull request merged**